airflow-core/src/airflow/ui/src/pages/TaskInstance/Logs/TaskLogHeader.tsx

```diff
@@ -178,20 +200,57 @@ export const TaskLogHeader = ({
             </Select.Content>
           </Select.Root>
         ) : undefined}
-        <HStack>
+        <HStack gap={1}>
           <Tooltip closeDelay={100} content="Press w to toggle wrap" openDelay={100}>
             <Button
+              _hover={{ bg: "gray.800" }}
               aria-label={wrap ? "Unwrap" : "Wrap"}
-              bg="bg.panel"
+              bg="black"
+              color="white"
+              m={0}
               onClick={toggleWrap}
+              px={4}
+              py={2}
               variant="outline"
             >
               {wrap ? "Unwrap" : "Wrap"}
             </Button>
           </Tooltip>
+          {toggleExpanded ? (
+            <Tooltip
+              closeDelay={100}
+              content={expanded ? "Press e to collapse" : "Press e to expand"}
+              openDelay={100}
+            >
+              <Button
+                _hover={{ bg: "gray.800" }}
+                aria-label={expanded ? "Collapse All" : "Expand"}
+                bg="black"
+                color="white"
+                m={0}
+                onClick={toggleExpanded}
+                px={4}
+                py={2}
+                size="md"
+                variant="outline"
+              >
+                {expanded ? "Collapse All" : "Expand "}
+              </Button>
+            </Tooltip>
+          ) : undefined}
           {!isFullscreen && (
             <Tooltip closeDelay={100} content="Press f for fullscreen" openDelay={100}>
-              <IconButton aria-label="Full screen" bg="bg.panel" onClick={toggleFullscreen} variant="outline">
+              <IconButton
+                _hover={{ bg: "gray.800" }}
```

Comment:
Let's use the semantic colors here. 
```suggestion
                _hover={{ bg: "bg.emphasized" }}
```

Reply:
Also changed the warp buton to this houver style